spec/feature/confirm_email_prompt_spec.rb

```diff
@@ -0,0 +1,111 @@
+require "pry"
+RSpec.feature "Confirm email prompt" do
+  scenario "Seeing the banner on new user signup" do
+    given_i_newly_created_an_account
```

Comment:
Could this be put into a global let block, e.g. `let!(:user) { FactoryBot.create(:user) }`, as we have in the `login_spec`?  It might reduce some of the duplication in each test.

Reply:
[How about this](https://github.com/alphagov/govuk-account-manager-prototype/pull/436/commits/704c904ca1113349ca31356adf845b8e92b6a43f), a mix of Factory traits, scenario blocks and lets